packages/react-components/src/icons/src/EmbeddedIcon.tsx

```diff
@@ -0,0 +1,38 @@
+import { Children, ReactElement, ReactNode } from "react";
+import { augmentElement, createSizeAdapter } from "../../shared";
+
+export interface EmbeddedIconProps {
+    /**
+     * An icon can vary in size.
+     */
+    size?: "2xs" | "xs" | "sm" | "md" | "lg" | "xl" | "inherit";
+    /**
+     * React children.
+     */
+    children: ReactNode;
+    /**
+     * Allow any extra props
+     */
+    [key: string]: any
```

Comment:
first time we have to specify this, since it's the only component so far that is not using the forwardRef property

Reply:
It's all good with me, but i wanted to show you